src/crimson/common/chained_dispatchers.h

```diff
@@ -0,0 +1,46 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:nil -*- 
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <deque>
+#include <boost/intrusive/slist.hpp>
+
+#include "crimson/net/Dispatcher.h"
+#include "crimson/common/log.h"
+
+using Dispatcher = crimson::net::Dispatcher;
```

Comment:
could just put
```c++
using crimson::net::Dispatcher;
```

but i think this defeat the purpose of using naming space by `using` in header file and in global namespace.